src/dbus/rauc/system_conf.rs

```diff
@@ -0,0 +1,161 @@
+use std::fmt::Write;
+use std::fs::{create_dir_all, read_to_string, remove_file, rename, write};
+use std::io::{Error, ErrorKind};
+use std::path::Path;
+
+use super::Channel;
+
+use log::info;
+
+#[cfg(feature = "demo_mode")]
+mod imports {
+    pub(super) const STATIC_CONF_PATH: &str = "demo_files/usr/lib/rauc/system.conf";
+    pub(super) const DYNAMIC_CONF_PATH: &str = "demo_files/run/rauc/system.conf";
+}
+
+#[cfg(not(feature = "demo_mode"))]
+mod imports {
+    pub(super) const STATIC_CONF_PATH: &str = "/usr/lib/rauc/system.conf";
+    pub(super) const DYNAMIC_CONF_PATH: &str = "/run/rauc/system.conf";
+}
+
+use imports::*;
+
+const MAGIC_LINE: &str = "\n# <tacd-poll-section>\n";
+
+fn poll_section(
+    primary_channel: Option<&Channel>,
+    polling: bool,
+    auto_install: bool,
+) -> Result<Option<String>, std::fmt::Error> {
+    // If no primary channel is configured or if polling is not enabled,
+    // then we do not need a `[poll]` section at all.
+    let primary_channel = match (primary_channel, polling) {
+        (Some(pc), true) => pc,
+        _ => return Ok(None),
+    };
+
+    let mut section = String::new();
+
+    writeln!(&mut section)?;
+    writeln!(&mut section, "[poll]")?;
```

Comment:
We've decided to rename the section from `poll` to `polling`.
```suggestion
    writeln!(&mut section, "[polling]")?;
```
Also update the comment above and in `update_system_conf()`.